lib/src/main/java/com/orange/ods/compose/component/progressindicator/OdsLinearProgressIndicator.kt

```diff
@@ -35,20 +49,88 @@ import com.orange.ods.utilities.extension.orElse
 @OdsComponentApi
 fun OdsLinearProgressIndicator(
     modifier: Modifier = Modifier,
+    label: String? = null,
+    currentValue: String? = null,
```

Comment:
I think the `currentValue` should be removed and replaced by `showCurrentValue`, a boolean parameter to know if we have to display the current value or not.
The current value should change according to the `progress` parameter value. So, you have to transform the progress value to a percentage value to display.

This change will have several impacts in your developments. Don't forget to remove strings like "70%" etc.